### DISCUSSION TITLE
Contradiction

### DISCUSSION THREAD

**Discussion by angelxaces** [awesome-windows11/windows11#19] [General]
You wanted to prove that windows could be customized without a third-party app.... So you gathered all the individual functions and made the bundle available as a third party without compiling them into a singular interface.... 

Don't get me wrong I think I'm going to try some of this out. Which is to say that is was a useful idea as far a repositories are concerned. I'm just not feeling like it accomplishes the point you were trying to make with it. If anything I get the impression you simply have something against UI within that of an OS. If that's the case then you have my empathy. To be so anti-UI would suggest that UI has caused trauma in your past. I too, recall the horrors of Windows 3.1 and 28.8k Fax Modems, before Dial-Up Internet and E-Mail... I still get nightmares.

**Comment by jestxfot** [comment]
I love linux, and I love the command line. This repository was originally meant to be a purely self-service repository to keep all the customizations in one place for quick access. However, it became popular by happy accident.  I have nothing against UI, however my philosophy does not like third party programs that do not clearly affect the system. I just like to control what I run and look for information on my own. Maybe that's the reason why the repository became popular.
